packages/vue-query/src/useQueries.ts

```diff
@@ -146,7 +148,16 @@ export function useQueries<T extends any[]>({
   const optionsQueryClient = unreffedQueries.value[0]?.queryClient as
     | QueryClient
     | undefined
-  const queryClient = optionsQueryClient ?? useQueryClient(queryClientKey)
+  const queryClient =
+    queryClientInjected ?? optionsQueryClient ?? useQueryClient(queryClientKey)
+  if (queryClientKey || optionsQueryClient) {
+    queryClient
+      .getLogger()
+      .error(
+        `Providing queryClient to individual queries in useQueries has been deprecated and will be removed in the next major version. You can still pass queryClient as an option directly to useQueries hook.`,
+      )
+  }
```

Comment:
can we put that behind a `if (process.env.NODE_ENV !== 'production') {` check? This is how we usually do those logs:

https://github.com/TanStack/query/blob/1e0745a6cc80cbdbafc3153812b7982a8f549447/packages/query-core/src/queryClient.ts#L76-L80

Reply:
Good catch!